src/main/webapp/directory.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Directory</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <p align="right"><button onclick="location.href='addevent.html';">Add Event</button></p>
+
+      <h1>Happening in &lt;School Name&gt;</h1>
+
+      <form action="/filter" method="POST">
+        <table style="width:100%; text-align:left;">
+          <tr>
+            <th>Event type:</th>
+            <th>Subject:</th>
+            <th>Sort by:</th>
+          </tr>
+          <tr style="vertical-align:top;">
+            <td>
+              <input type="checkbox" id="social" name="social" value="social">
+              <label for="social">Social</label>
+              <br>
+              <input type="checkbox" id="study" name="study" value="study">
+              <label for="study">Study</label>
+            </td>
+            <td>
+                <input type="checkbox" id="english" name="english" value="english">
+                <label for="english">English</label>
+                <br>
+                <input type="checkbox" id="history" name="history" value="history">
+                <label for="history">History</label>
+                <br>
+                <input type="checkbox" id="science" name="science" value="science">
+                <label for="science">Science</label>
+                <br>
+                <input type="checkbox" id="math" name="math" value="math">
+                <label for="math">Math</label>
+            </td>
+            <td>
+              <select>
+                <option value="time">Time</option>
+                <option value="location">Location</option>
+              </select>
+            </td>
+          </tr>
+        </table>
+        <br>
+        <input type="submit" value="Filter"/>
+      </form>
+      <br>
+
+      <ul id="directory-list">
+        <li>
```

Comment:
For future reference: One thing that you also mentioned in the team meeting is setting a limit on how many items are displayed initially and do something like continuous scrolling.

I think maybe what we can do to start off is display maybe 5 events or so, then add a "Show More" or "Expand All" button which will display more events for the user. Then if we have time leftover, we can look into continuous scrolling. What do you think?

Reply:
Yeah, we could keep loading N more events at a time, or having 'previous' and 'next' buttons. I think we could start off by just loading all events for the simplest implementation and then address this.